test/integration/ProjectBrowserTest.kt

```diff
@@ -16,66 +16,65 @@
  *
  */
 
-// We need to access private function Studio.MainWindowColumn, this allows us to.
-// Do not use this outside of tests anywhere. It is extremely dangerous to do so.
-@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
-
 package com.vaticle.typedb.studio.test.integration
 
-import androidx.compose.ui.test.onNodeWithText
-import androidx.compose.ui.test.performClick
+import com.vaticle.typedb.studio.framework.material.Icon
 import com.vaticle.typedb.studio.state.StudioState
-import com.vaticle.typedb.studio.test.integration.Utils.cloneAndOpenProject
-import com.vaticle.typedb.studio.test.integration.Utils.studioTest
-import com.vaticle.typedb.studio.test.integration.Utils.delayAndRecompose
-import com.vaticle.typedb.studio.test.integration.Utils.SAMPLE_DATA_PATH
-import com.vaticle.typedb.studio.test.integration.Utils.DOUBLE_CHEVRON_DOWN_ICON_STRING
-import com.vaticle.typedb.studio.test.integration.Utils.DOUBLE_CHEVRON_UP_ICON_STRING
+import com.vaticle.typedb.studio.test.integration.common.Paths.SAMPLE_DATA_PATH
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.clickIcon
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.createData
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.delayAndRecompose
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.nodeWithTextDoesNotExist
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.nodeWithTextExists
+import com.vaticle.typedb.studio.test.integration.common.StudioActions.openProject
+import kotlinx.coroutines.runBlocking
 import org.junit.Test
 
 class ProjectBrowserTest: IntegrationTest() {
 
     @Test
     fun createADirectory() {
-        studioTest(composeRule) {
+        runBlocking {
             val createdDirectoryName = "created"
 
-            cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
+            createData(source = SAMPLE_DATA_PATH, destination = testID)
+            openProject(composeRule, testID)
 
             StudioState.project.current!!.directory.asDirectory().tryCreateDirectory(createdDirectoryName)
             delayAndRecompose(composeRule)
 
             StudioState.project.current!!.reloadEntries()
             delayAndRecompose(composeRule)
 
-            composeRule.onNodeWithText(createdDirectoryName).assertExists()
+            nodeWithTextExists(composeRule, createdDirectoryName)
```

Comment:
1. `assert` name is critical to describing functionality, but is missing
2. `exists` is more important than `withText`
3. `createdDirectoryName`: let's be very clear about what it's used for via named param
```suggestion
            assertNodeExistsWithText(composeRule, text = createdDirectoryName)
```

Reply:
Agreed, implemented.